docs/test_rst.rst

```diff
@@ -0,0 +1,5 @@
+========
```

Comment:
the preceding `========` line is actually unnecessary. This title will render exactly the same without it. Titles only need a `==========` (like `##` in MD) or `----------------` (like `#` in MD) line after the title in a length of at least the title's length. A side note: The difference between the two aforementioned example lines is used to nest sections.